src/Emulator.cpp

```diff
@@ -1339,6 +1389,47 @@ void TEmulator::InsertRomModul(bool inserted)
 	delete[] buff;
 }
 //---------------------------------------------------------------------------
+void TEmulator::InsertMegaRomModul(bool inserted)
```

Comment:
```suggestion
void TEmulator::InsertMegaRomModule(bool inserted)
```
Typo, but also fix `InsertRomModul` -> `InsertRomModule`, please 🙏 😊

Reply:
sure, I will change both names to end with ...Module()